src/lib/Containers/Inquiry.tsx

```diff
@@ -0,0 +1,219 @@
+  align-items: center
+  min-height: 12
+  margin-top: 5
+`
+
+export class Inquiry extends React.Component<RelayProps, any> {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      text: this.props.inquiryArtwork.contact_message,
+      sending: false,
+    }
+  }
+
+  dismissModal() {
+    ARSwitchBoard.dismissModalViewController(this)
+  }
+
+  sendInquiry() {
+    // Using setState to trigger re-render for the button
+    this.setState(previousState => {
+      return { sending: true }
+    })
+    const { Emission } = NativeModules
+    fetch(gravityURL + "/api/v1/me/artwork_inquiry_request", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+        "X-ACCESS-TOKEN": Emission.authenticationToken,
+      },
+      body: JSON.stringify({
+        artwork: this.props.inquiryArtwork.id,
+        message: this.state.text,
+      }),
+    })
+      .then(response => {
+        console.log(response)
+        if (response.status >= 200 && response.status < 300) {
+          this.dismissModal()
+        } else {
+          this.setState(previousState => {
+            return { sending: false }
+          })
+          const error = new NetworkError(response.statusText)
+          console.log(error.message)
+          error.response = response
+          throw error
+        }
+      })
+      .catch(error => {
+        this.setState(previousState => {
+          return { sending: false }
+        })
+        console.log(error.message)
+        console.log(error)
+        throw error
+      })
+  }
+
+  render() {
+    const message = this.state.text
+    const partnerResponseRate = "2 DAY RESPONSE TIME" // currently hardcoded
+    const inquiryArtwork = this.props.inquiryArtwork
+    const partnerName = this.props.inquiryArtwork.partner.name
+    const buttonText = this.state.sending ? "SENDING..." : "SEND"
+
+    const doneButtonStyles = {
+      backgroundColor: colors["purple-regular"],
+      marginBottom: 0,
+      paddingTop: 12,
+      height: 44,
+    }
+
+    return (
+      <Container>
+        <StatusBar />
+        <BottomAlignedButton
+          onPress={this.sendInquiry.bind(this)}
+          bodyStyle={doneButtonStyles}
+          buttonText={buttonText}
+          disabled={this.state.sending}
+        >
+          <Header>
+            <HeaderTextContainer>
+              <CancelButton onPress={this.dismissModal.bind(this)}><MetadataText>CANCEL</MetadataText></CancelButton>
+              <TitleView>
+                <SmallHeadline>{partnerName}</SmallHeadline>
+                <ResponseRateLine>
+                  <ResponseIndicator />
+                  <ResponseRate>{partnerResponseRate}</ResponseRate>
+                </ResponseRateLine>
+              </TitleView>
+              <PlaceholderView>CANCEL</PlaceholderView>
+            </HeaderTextContainer>
+          </Header>
+          <Content>
+            <ArtworkPreview artwork={inquiryArtwork} />
```

Comment:
Right, this error is legit, because your component and the `ArtworkPreview` component use different props. However, fetching those props is left as a detail to Relay, so you should treat it as an opaque object. i.e. `<ArtworkPreview artwork={inquiryArtwork as any} />` is correct